system-int-test/test/cmr/system_int_test/search/autocomplete_collection_facet_search_test.clj

```diff
@@ -78,7 +78,7 @@
      (is (:CMR-Took headers))))
 
   (testing "entries return in descending score order"
-   (let [response (query->json-response-body "q=at")
+   (let [response (query->json-response-body "q=gulf")
```

Comment:
why do we change this test?

Reply:
changed by mistake, restored to previous.